docs/tutorial/node/src/getting_started.md

```diff
@@ -1,48 +1,45 @@
 <!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
 # Getting Started
 
-In this tutorial, we use JavaScript as the host programming language. We also support other programming languages like
+This tutorial uses JavaScript as the host programming language. Slint also supports other programming languages like
 [Rust](https://slint.dev/docs/rust/slint/) or [C++](https://slint.dev/docs/cpp/).
 
-You'll need a development environment with [Node.js 16](https://nodejs.org/download/release/v16.19.1/) and [npm](https://www.npmjs.com/) installed. More recent
-versions of NodeJS are currently not supported, for details check [Issue #961](https://github.com/slint-ui/slint/issues/961).
-Since Slint is implemented in the Rust programming language, you also need to install a Rust compiler (1.70 or newer). You can easily install a Rust compiler
-following the instruction from [the Rust website](https://www.rust-lang.org/learn/get-started).
-You will also need some additional platform-specific dependencies, see <https://github.com/slint-ui/slint/blob/master/docs/building.md#prerequisites>
+Slint has an application template you can use to create a project with dependencies already set up that follows recommended best practices.
 
-We're going to use `slint-ui` as `npm` dependency.
+As Slint is implemented in the Rust programming language, you also need to install a Rust compiler (1.70 or newer). You can install a Rust compiler
```

Comment:
Unrelated to your cleanup, but an opportunity: an installation of Rust is not needed anymore (in the most common case). I'd say it would be safe to omit that. Thoughts @FloVanGH @ogoffart ?

Reply:
@tronical I just uninstalled Rust and things did seem to work, so i think it's OK to remove.